app/views/obituaries/show.html.erb

```diff
@@ -15,24 +15,26 @@
       <h3 class="my-3">Obituary</h3>
       <p><%= @obituary.description %></p>
       <h3 class="my-3">Details</h3>
-      <ul>
-        <li>Age: <%= @obituary.age %></li>
-        <li>City: <%= @obituary.city %></li>
-        <li>State: <%= @obituary.state %></li>
+      <ul class="no-bullets">
+        <li><b>Age: </b><%= @obituary.age %></li>
+        <li><b>Location: </b><%= @obituary.location %></li>
+        <li><b>Date Published: </b><%= @obituary.date %></li>
         <% if @obituary.covid == true %>
-          <li><%= "This person died from COVID-19." %></li>
+          <br><li><i><%= "This person died from COVID-19." %></i></li>
+          <li><%= link_to "Read more obituaries of people who died from COVID-19.", obituaries_covid_19_path %></li>
         <% end %>
       </ul>
       <br>
+      <% if current_user && current_user.has_obituary?(@obituary) %>
+        <button class="btn btn-outline-secondary mr-1"><%= link_to 'Edit', edit_obituary_path(@obituary.id) %></button>
+        <button class="btn btn-outline-secondary mr-1"><%= link_to 'Delete', "/obituaries/#{@obituary.id}", method: :delete %></button>
```

Comment:
The buttons are the same style as the profile, add obituary, and logout buttons since they all seemed to have in common that they only appear for logged in users